File: omnic-v2/test_export.js
let VERSION = "1.0.0";
function greet(name) {
    return "Hello, ";
    // Unknown stmt kind: 0
    name;
}
class Person {
    constructor(data = {}) {
        this.name = data.name;
        this.age = data.age;
    }
}
function internal_helper() {
    return 42;
}

module.exports = { VERSION, greet, Person };

File: omnic-v2/test_export.py
VERSION = "1.0.0"
def greet(name):
    return "Hello, "
    # Unknown stmt: 0
    name

class Person:
    def __init__(self, data=None):
        if data is None: data = {}
        self.name = data.get('name')
        self.age = data.get('age')


def internal_helper():
    return 42


__all__ = ['VERSION', 'greet', 'Person']
